frame/election-provider-support/src/lib.rs

```diff
@@ -294,3 +294,69 @@ impl<AccountId, BlockNumber> ElectionProvider<AccountId, BlockNumber> for () {
 		Err("<() as ElectionProvider> cannot do anything.")
 	}
 }
+
+/// A utility trait for something to implement `ElectionDataProvider` in a sensible way.
+///
+/// This is generic over `AccountId` and it can represent a validator, a nominator, or any other
+/// entity.
+///
+/// To simplify the trait, the `VoteWeight` is hardcoded as the weight of each
```

Comment:
line wrapping is a bit broken here.

Reply:
fa674aa968c7927001be0d7c10fbeecc65f25a00